CRITICAL - 2022-12-22 05:26:02 --> Cannot declare class App\Models\master\rkas_m, because the name is already in use
in APPPATH\Models\report\rkas_m.php on line 7.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2022-12-22 05:26:36 --> Cannot declare class App\Models\master\rkas_m, because the name is already in use
in APPPATH\Models\report\rkas_m.php on line 7.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2022-12-22 05:26:46 --> Cannot declare class App\Models\master\rkas_m, because the name is already in use
in APPPATH\Models\report\rkas_m.php on line 7.
 1 [internal function]: CodeIgniter\Debug\Exceptions->shutdownHandler()
CRITICAL - 2022-12-22 05:27:08 --> Invalid file: report/rkas_v.php
in SYSTEMPATH\Exceptions\FrameworkException.php on line 33.
 1 SYSTEMPATH\View\View.php(201): CodeIgniter\Exceptions\FrameworkException::forInvalidFile('report/rkas_v.php')
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rkas_v', [], true)
 3 APPPATH\Controllers\report\rkas.php(23): view('report/rkas_v', [...])
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rkas->index()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rkas))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 05:27:30 --> mysqli_sql_exception: Unknown column 'kas_name' in 'order clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\pos\app\Views\report\rkas_v.php(117): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\View\View.php(213): include('D:\\xampp\\htdocs...')
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Common.php(1134): CodeIgniter\View\View->render('report/rkas_v', Array, true)
#8 D:\xampp\htdocs\pos\app\Controllers\report\rkas.php(23): view('report/rkas_v', Array)
#9 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\report\rkas->index()
#10 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rkas))
#11 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2022-12-22 05:27:30 --> Unknown column 'kas_name' in 'order clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *
FROM `kas`
LEFT JOIN `store` ON `store`.`store_id`=`kas`.`store_id`
WHERE `kas`.`store_id` = \'0\'
ORDER BY `kas_name` ASC', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `kas`
LEFT JOIN `store` ON `store`.`store_id`=`kas`.`store_id`
WHERE `kas`.`store_id` = \'0\'
ORDER BY `kas_name` ASC')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `kas`
LEFT JOIN `store` ON `store`.`store_id`=`kas`.`store_id`
WHERE `kas`.`store_id` = \'0\'
ORDER BY `kas_name` ASC')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `kas`
LEFT JOIN `store` ON `store`.`store_id`=`kas`.`store_id`
WHERE `kas`.`store_id` = :kas.store_id:
ORDER BY `kas_name` ASC', [...], false)
 5 APPPATH\Views\report\rkas_v.php(117): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rkas_v.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rkas_v', [], true)
 9 APPPATH\Controllers\report\rkas.php(23): view('report/rkas_v', [...])
10 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rkas->index()
11 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rkas))
12 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 06:59:12 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 06:59:12 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 06:59:16 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 06:59:16 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 06:59:17 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 06:59:17 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 06:59:17 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 06:59:17 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 06:59:17 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 06:59:17 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 06:59:19 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 06:59:19 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'45000\', \'50000\', \'5000\', 0, NULL, \'45000\', \'masuk\', \'Pembayaran POS202212190945500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 06:59:25 --> Attempt to read property "product_sell" on null
in APPPATH\Controllers\transaction\transaction.php on line 329.
 1 APPPATH\Controllers\transaction\transaction.php(329): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Attempt to read property "product_sell" on null', 'APPPATH\\Controllers\\transaction\\transaction.php', 329)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:01:17 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:01:17 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'17000\', \'17000\', \'0\', 0, NULL, \'17000\', \'masuk\', \'Pembayaran POS202212192121360\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'17000\', \'17000\', \'0\', 0, NULL, \'17000\', \'masuk\', \'Pembayaran POS202212192121360\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'17000\', \'17000\', \'0\', 0, NULL, \'17000\', \'masuk\', \'Pembayaran POS202212192121360\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:02:43 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:02:43 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192121480\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192121480\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192121480\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:03:03 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:03:03 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192121500\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192121500\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192121500\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:03:21 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:03:21 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192122120\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192122120\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212192122120\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:06:06 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:06:06 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'9000\', \'\', \'-9000\', 0, NULL, \'9000\', \'masuk\', \'Pembayaran POS202212220706000\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'9000\', \'\', \'-9000\', 0, NULL, \'9000\', \'masuk\', \'Pembayaran POS202212220706000\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'9000\', \'\', \'-9000\', 0, NULL, \'9000\', \'masuk\', \'Pembayaran POS202212220706000\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:14:10 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:14:10 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220713180\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220713180\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220713180\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:14:27 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:14:27 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220714150\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220714150\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220714150\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:15:20 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:15:20 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220714420\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220714420\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, NULL, \'0\', \'masuk\', \'Pembayaran POS202212220714420\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:15:31 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:15:31 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, \'2\', \'0\', \'masuk\', \'Pembayaran POS202212220715240\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, \'2\', \'0\', \'masuk\', \'Pembayaran POS202212220715240\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'0\', \'\', \'0\', 0, \'2\', \'0\', \'masuk\', \'Pembayaran POS202212220715240\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:19:08 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:19:08 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'8000\', \'8000\', \'0\', 0, \'2\', \'8000\', \'masuk\', \'Pembayaran POS202212220716540\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'8000\', \'8000\', \'0\', 0, \'2\', \'8000\', \'masuk\', \'Pembayaran POS202212220716540\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'8000\', \'8000\', \'0\', 0, \'2\', \'8000\', \'masuk\', \'Pembayaran POS202212220716540\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 07:19:30 --> mysqli_sql_exception: Unknown column 'transaction_bill' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 07:19:30 --> Unknown column 'transaction_bill' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'8000\', \'8000\', \'0\', 0, \'2\', \'8000\', \'masuk\', \'Pembayaran POS202212220716540\', \'2022-12-22\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'8000\', \'8000\', \'0\', 0, \'2\', \'8000\', \'masuk\', \'Pembayaran POS202212220716540\', \'2022-12-22\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (\'8000\', \'8000\', \'0\', 0, \'2\', \'8000\', \'masuk\', \'Pembayaran POS202212220716540\', \'2022-12-22\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `kas` (`transaction_bill`, `transaction_pay`, `transaction_change`, `transaction_status`, `kas_shift`, `kas_nominal`, `kas_type`, `kas_description`, `kas_date`) VALUES (:transaction_bill:, :transaction_pay:, :transaction_change:, :transaction_status:, :kas_shift:, :kas_nominal:, :kas_type:, :kas_description:, :kas_date:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(456): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->pelunasan()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 08:07:38 --> Undefined property: stdClass::$transaction_shift
in APPPATH\Views\report\rtransaction_v.php on line 58.
 1 APPPATH\Views\report\rtransaction_v.php(58): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$transaction_shift', 'APPPATH\\Views\\report\\rtransaction_v.php', 58)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rtransaction_v.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransaction_v', [], true)
 5 APPPATH\Controllers\report\rtransaction.php(23): view('report/rtransaction_v', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransaction->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 08:11:54 --> Undefined property: stdClass::$transaction_description
in APPPATH\Views\report\rtransaction_v.php on line 59.
 1 APPPATH\Views\report\rtransaction_v.php(59): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$transaction_description', 'APPPATH\\Views\\report\\rtransaction_v.php', 59)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rtransaction_v.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransaction_v', [], true)
 5 APPPATH\Controllers\report\rtransaction.php(23): view('report/rtransaction_v', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransaction->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:33 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:33 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831330\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831330\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831330\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:38 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:38 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:38 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:38 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:38 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:39 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831380\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:45 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:45 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831450\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831450\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831450\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:45 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:45 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831450\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831450\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831450\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:46 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:46 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831460\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831460\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831460\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:31:46 --> mysqli_sql_exception: Column 'transaction_shift' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 08:31:46 --> Column 'transaction_shift' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831460\', \'1\', \'0\', 2, NULL)', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831460\', \'1\', \'0\', 2, NULL)')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (\'2022-12-22\', \'POS202212220831460\', \'1\', \'0\', 2, NULL)')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transaction` (`transaction_date`, `transaction_no`, `cashier_id`, `store_id`, `transaction_status`, `transaction_shift`) VALUES (:transaction_date:, :transaction_no:, :cashier_id:, :store_id:, :transaction_status:, :transaction_shift:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(310): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->createnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 08:45:11 --> Undefined property: CodeIgniter\HTTP\IncomingRequest::$getGet
in APPPATH\Views\report\rtransaction_v.php on line 62.
 1 APPPATH\Views\report\rtransaction_v.php(62): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: CodeIgniter\\HTTP\\IncomingRequest::$getGet', 'APPPATH\\Views\\report\\rtransaction_v.php', 62)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rtransaction_v.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransaction_v', [], true)
 5 APPPATH\Controllers\report\rtransaction.php(23): view('report/rtransaction_v', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransaction->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 08:46:03 --> Undefined property: CodeIgniter\HTTP\IncomingRequest::$getGet
in APPPATH\Views\report\rtransaction_v.php on line 65.
 1 APPPATH\Views\report\rtransaction_v.php(65): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: CodeIgniter\\HTTP\\IncomingRequest::$getGet', 'APPPATH\\Views\\report\\rtransaction_v.php', 65)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rtransaction_v.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransaction_v', [], true)
 5 APPPATH\Controllers\report\rtransaction.php(23): view('report/rtransaction_v', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransaction->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 08:50:14 --> mysqli_sql_exception: Unknown column 'transactiond.cashier_id' in 'on clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *\nFROM `...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *\nFROM `...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *\nFROM `...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *\nFROM `...', Array, false)
#4 D:\xampp\htdocs\pos\app\Views\report\rtransactiond_v.php(69): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\View\View.php(213): include('D:\\xampp\\htdocs...')
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Common.php(1134): CodeIgniter\View\View->render('report/rtransac...', Array, true)
#8 D:\xampp\htdocs\pos\app\Controllers\report\rtransactiond.php(23): view('report/rtransac...', Array)
#9 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\report\rtransactiond->index()
#10 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransactiond))
#11 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#12 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#13 {main}
CRITICAL - 2022-12-22 08:50:14 --> Unknown column 'transactiond.cashier_id' in 'on clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT *
FROM `transactiond`
LEFT JOIN `store` ON `store`.`store_id`=`transactiond`.`store_id`
LEFT JOIN `user` ON `user`.`user_id`=`transactiond`.`cashier_id`
WHERE `transactiond`.`store_id` = \'0\'
ORDER BY `transactiond_id` DESC', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT *
FROM `transactiond`
LEFT JOIN `store` ON `store`.`store_id`=`transactiond`.`store_id`
LEFT JOIN `user` ON `user`.`user_id`=`transactiond`.`cashier_id`
WHERE `transactiond`.`store_id` = \'0\'
ORDER BY `transactiond_id` DESC')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT *
FROM `transactiond`
LEFT JOIN `store` ON `store`.`store_id`=`transactiond`.`store_id`
LEFT JOIN `user` ON `user`.`user_id`=`transactiond`.`cashier_id`
WHERE `transactiond`.`store_id` = \'0\'
ORDER BY `transactiond_id` DESC')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT *
FROM `transactiond`
LEFT JOIN `store` ON `store`.`store_id`=`transactiond`.`store_id`
LEFT JOIN `user` ON `user`.`user_id`=`transactiond`.`cashier_id`
WHERE `transactiond`.`store_id` = :transactiond.store_id:
ORDER BY `transactiond_id` DESC', [...], false)
 5 APPPATH\Views\report\rtransactiond_v.php(69): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rtransactiond_v.php')
 7 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 8 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransactiond_v', [], true)
 9 APPPATH\Controllers\report\rtransactiond.php(23): view('report/rtransactiond_v', [...])
10 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransactiond->index()
11 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransactiond))
12 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
13 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 08:53:27 --> syntax error, unexpected token "->", expecting end of file
in APPPATH\Views\report\rtransactiond_v.php on line 57.
 1 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 2 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransactiond_v', [], true)
 3 APPPATH\Controllers\report\rtransactiond.php(23): view('report/rtransactiond_v', [...])
 4 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransactiond->index()
 5 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransactiond))
 6 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 7 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 08:53:49 --> Undefined property: stdClass::$transactiond_no
in APPPATH\Views\report\rtransactiond_v.php on line 73.
 1 APPPATH\Views\report\rtransactiond_v.php(73): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$transactiond_no', 'APPPATH\\Views\\report\\rtransactiond_v.php', 73)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rtransactiond_v.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rtransactiond_v', [], true)
 5 APPPATH\Controllers\report\rtransactiond.php(23): view('report/rtransactiond_v', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rtransactiond->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rtransactiond))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 09:13:30 --> mysqli_sql_exception: Column 'transaction_id' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(352): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 09:13:30 --> Column 'transaction_id' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (:transaction_id:, :product_id:, :store_id:, :transactiond_qty:, :transactiond_price:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 09:13:31 --> mysqli_sql_exception: Column 'transaction_id' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(352): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 09:13:31 --> Column 'transaction_id' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (:transaction_id:, :product_id:, :store_id:, :transactiond_qty:, :transactiond_price:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:39 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:41 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:41 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:43 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:45 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:53 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:53 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:26:55 --> Undefined variable $product
in APPPATH\Controllers\transaction\transaction.php on line 735.
 1 APPPATH\Controllers\transaction\transaction.php(735): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $product', 'APPPATH\\Controllers\\transaction\\transaction.php', 735)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->listproductlist()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 09:28:39 --> mysqli_sql_exception: Column 'transaction_id' cannot be null in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `tr...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `tr...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `tr...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `tr...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(352): CodeIgniter\Database\BaseBuilder->insert(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 09:28:39 --> Column 'transaction_id' cannot be null
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (NULL, \'6\', \'0\', 1, \'8000\')')
 4 SYSTEMPATH\Database\BaseBuilder.php(1916): CodeIgniter\Database\BaseConnection->query('INSERT INTO `transactiond` (`transaction_id`, `product_id`, `store_id`, `transactiond_qty`, `transactiond_price`) VALUES (:transaction_id:, :product_id:, :store_id:, :transactiond_qty:, :transactiond_price:)', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Database\BaseBuilder->insert([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 09:46:04 --> mysqli_sql_exception: Unknown column 'product_qty' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `product...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `product...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(353): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 09:46:04 --> Unknown column 'product_qty' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `product` SET `product_qty` = 2
WHERE `product_id` = \'8\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product` SET `product_qty` = 2
WHERE `product_id` = \'8\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product` SET `product_qty` = 2
WHERE `product_id` = \'8\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `product` SET `product_qty` = :product_qty:
WHERE `product_id` = :product_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(353): CodeIgniter\Database\BaseBuilder->update([...], [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 09:46:07 --> mysqli_sql_exception: Unknown column 'product_qty' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `product...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `product...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(353): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 09:46:07 --> Unknown column 'product_qty' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `product` SET `product_qty` = 3
WHERE `product_id` = \'8\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product` SET `product_qty` = 3
WHERE `product_id` = \'8\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product` SET `product_qty` = 3
WHERE `product_id` = \'8\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `product` SET `product_qty` = :product_qty:
WHERE `product_id` = :product_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(353): CodeIgniter\Database\BaseBuilder->update([...], [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 09:46:07 --> mysqli_sql_exception: Unknown column 'product_qty' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `product...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `product...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(353): CodeIgniter\Database\BaseBuilder->update(Array, Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 09:46:07 --> Unknown column 'product_qty' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('UPDATE `product` SET `product_qty` = 4
WHERE `product_id` = \'8\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('UPDATE `product` SET `product_qty` = 4
WHERE `product_id` = \'8\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('UPDATE `product` SET `product_qty` = 4
WHERE `product_id` = \'8\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2079): CodeIgniter\Database\BaseConnection->query('UPDATE `product` SET `product_qty` = :product_qty:
WHERE `product_id` = :product_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(353): CodeIgniter\Database\BaseBuilder->update([...], [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:50:03 --> Undefined property: stdClass::$product_qty
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_qty', 'APPPATH\\Controllers\\transaction\\transaction.php', 352)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:50:07 --> Undefined property: stdClass::$product_qty
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_qty', 'APPPATH\\Controllers\\transaction\\transaction.php', 352)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 09:50:08 --> Undefined property: stdClass::$product_qty
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_qty', 'APPPATH\\Controllers\\transaction\\transaction.php', 352)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:01:52 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::get()
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:01:54 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::get()
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:01:55 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::get()
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:01:56 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::get()
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:03:02 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::get()
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:03:04 --> Call to undefined method CodeIgniter\Database\MySQLi\Result::get()
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:03:52 --> Undefined property: stdClass::$product_qty
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_qty', 'APPPATH\\Controllers\\transaction\\transaction.php', 352)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 10:04:17 --> Undefined property: stdClass::$product_qty
in APPPATH\Controllers\transaction\transaction.php on line 352.
 1 APPPATH\Controllers\transaction\transaction.php(352): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_qty', 'APPPATH\\Controllers\\transaction\\transaction.php', 352)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->insertnota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 11:06:38 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::affectedRows()
in APPPATH\Controllers\transaction\transaction.php on line 410.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletetransactiond()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 11:11:25 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::affectedRows()
in APPPATH\Controllers\transaction\transaction.php on line 410.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletetransactiond()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 12:04:03 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::affectedRows()
in APPPATH\Controllers\transaction\transaction.php on line 410.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletetransactiond()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 12:11:02 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::affectedRows()
in APPPATH\Controllers\transaction\transaction.php on line 410.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletetransactiond()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 12:21:11 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::affectedRows()
in APPPATH\Controllers\transaction\transaction.php on line 410.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletetransactiond()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 12:21:20 --> Call to undefined method CodeIgniter\Database\MySQLi\Builder::affectedRows()
in APPPATH\Controllers\transaction\transaction.php on line 410.
 1 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletetransactiond()
 2 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 3 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 4 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 13:17:14 --> Undefined property: stdClass::$product_id
in APPPATH\Controllers\transaction\transaction.php on line 389.
 1 APPPATH\Controllers\transaction\transaction.php(389): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_id', 'APPPATH\\Controllers\\transaction\\transaction.php', 389)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 13:18:22 --> Undefined property: stdClass::$product_id
in APPPATH\Controllers\transaction\transaction.php on line 390.
 1 APPPATH\Controllers\transaction\transaction.php(390): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_id', 'APPPATH\\Controllers\\transaction\\transaction.php', 390)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 13:19:32 --> Undefined property: stdClass::$product_id
in APPPATH\Controllers\transaction\transaction.php on line 390.
 1 APPPATH\Controllers\transaction\transaction.php(390): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined property: stdClass::$product_id', 'APPPATH\\Controllers\\transaction\\transaction.php', 390)
 2 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 3 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 4 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 5 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:21:56 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:21:56 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 28
AND `transaction_id` = \'33\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 28
AND `transaction_id` = \'33\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 28
AND `transaction_id` = \'33\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:22:51 --> mysqli_sql_exception: Unknown column 'product_stock' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT `product...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `product...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `product...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(388): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:22:51 --> Unknown column 'product_stock' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = :transaction_id:
GROUP BY `product_id`', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(388): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:22:55 --> mysqli_sql_exception: Unknown column 'product_stock' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT `product...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `product...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `product...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(388): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:22:55 --> Unknown column 'product_stock' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = :transaction_id:
GROUP BY `product_id`', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(388): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:23:01 --> mysqli_sql_exception: Unknown column 'product_stock' in 'field list' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('SELECT `product...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `product...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `product...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `product...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(388): CodeIgniter\Database\BaseBuilder->get()
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:23:01 --> Unknown column 'product_stock' in 'field list'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = \'33\'
GROUP BY `product_id`')
 4 SYSTEMPATH\Database\BaseBuilder.php(1585): CodeIgniter\Database\BaseConnection->query('SELECT `product_id`, `product_stock`, SUM(transactiond_qty)AS qty
FROM `transactiond`
WHERE `transaction_id` = :transaction_id:
GROUP BY `product_id`', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(388): CodeIgniter\Database\BaseBuilder->get()
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:24:49 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:24:49 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 8
AND `transaction_id` = \'32\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 8
AND `transaction_id` = \'32\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 8
AND `transaction_id` = \'32\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:26:05 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:26:05 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'34\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'34\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'34\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:27:04 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:27:04 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'35\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'35\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'35\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:28:10 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:28:10 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 2
AND `transaction_id` = \'36\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 2
AND `transaction_id` = \'36\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 2
AND `transaction_id` = \'36\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 13:31:10 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 13:31:10 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'37\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'37\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 25
AND `transaction_id` = \'37\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(409): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
ERROR - 2022-12-22 16:01:05 --> mysqli_sql_exception: Unknown column 'product_stock' in 'where clause' in D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php:292
Stack trace:
#0 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `ka...', 0)
#1 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `ka...')
#2 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `ka...')
#3 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `ka...', Array, false)
#4 D:\xampp\htdocs\pos\app\Controllers\transaction\transaction.php(410): CodeIgniter\Database\BaseBuilder->delete(Array)
#5 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
#6 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
#7 D:\xampp\htdocs\pos\vendor\codeigniter4\framework\system\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(NULL, Object(Config\Cache), false)
#8 D:\xampp\htdocs\pos\public\index.php(67): CodeIgniter\CodeIgniter->run()
#9 {main}
CRITICAL - 2022-12-22 16:01:05 --> Unknown column 'product_stock' in 'where clause'
in SYSTEMPATH\Database\MySQLi\Connection.php on line 292.
 1 SYSTEMPATH\Database\MySQLi\Connection.php(292): mysqli->query('DELETE FROM `kas`
WHERE `product_stock` = 2
AND `transaction_id` = \'38\'', 0)
 2 SYSTEMPATH\Database\BaseConnection.php(666): CodeIgniter\Database\MySQLi\Connection->execute('DELETE FROM `kas`
WHERE `product_stock` = 2
AND `transaction_id` = \'38\'')
 3 SYSTEMPATH\Database\BaseConnection.php(593): CodeIgniter\Database\BaseConnection->simpleQuery('DELETE FROM `kas`
WHERE `product_stock` = 2
AND `transaction_id` = \'38\'')
 4 SYSTEMPATH\Database\BaseBuilder.php(2391): CodeIgniter\Database\BaseConnection->query('DELETE FROM `kas`
WHERE `product_stock` = :product_stock:
AND `transaction_id` = :transaction_id:', [...], false)
 5 APPPATH\Controllers\transaction\transaction.php(410): CodeIgniter\Database\BaseBuilder->delete([...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\transaction\transaction->deletenota()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\transaction\transaction))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
CRITICAL - 2022-12-22 16:57:13 --> Undefined variable $pemasukan
in APPPATH\Views\report\rlabarugi_v.php on line 38.
 1 APPPATH\Views\report\rlabarugi_v.php(38): CodeIgniter\Debug\Exceptions->errorHandler(2, 'Undefined variable $pemasukan', 'APPPATH\\Views\\report\\rlabarugi_v.php', 38)
 2 SYSTEMPATH\View\View.php(213): include('APPPATH\\Views\\report\\rlabarugi_v.php')
 3 SYSTEMPATH\View\View.php(216): CodeIgniter\View\View->CodeIgniter\View\{closure}()
 4 SYSTEMPATH\Common.php(1134): CodeIgniter\View\View->render('report/rlabarugi_v', [], true)
 5 APPPATH\Controllers\report\rlabarugi.php(23): view('report/rlabarugi_v', [...])
 6 SYSTEMPATH\CodeIgniter.php(927): App\Controllers\report\rlabarugi->index()
 7 SYSTEMPATH\CodeIgniter.php(482): CodeIgniter\CodeIgniter->runController(Object(App\Controllers\report\rlabarugi))
 8 SYSTEMPATH\CodeIgniter.php(351): CodeIgniter\CodeIgniter->handleRequest(null, Object(Config\Cache), false)
 9 FCPATH\index.php(67): CodeIgniter\CodeIgniter->run()
